test/test_helper.rb

```diff
@@ -29,7 +32,7 @@
 
 require 'minitest/reporters'
 Minitest::Reporters.use! [
-  Minitest::Reporters::ProgressReporter.new,
+  !ENV['TIME'] ? Minitest::Reporters::ProgressReporter.new : nil,
   ENV['HTML'] && Minitest::Reporters::HtmlReporter.new, # Create an HTML report with more information
   ENV['TIME'] && Minitest::Reporters::SpecReporter.new, # Generate a report to find slowest tests
```

Comment:
If they are mutually exclusive, it could be better to show it explicitly 
```suggestion
  ENV['TIME'] ? Minitest::Reporters::SpecReporter.new : Minitest::Reporters::ProgressReporter.new , # Generate a report to find slowest tests
```